apps/admin-page/src/app/app.component.ts

```diff
@@ -7,4 +7,10 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'admin-page';
+  public sidebarCollapsed = false;
+  public username = 'username@example.com';
+
+  toggleCollapsed() {
```

Comment:
Add a return type to the method. Right now the linter doesn't point it out since I forgot to force that rule, but it will be enforced via the linter 🔜 
```suggestion
  toggleCollapsed(): void {
```

The method is also missing JsDoc (which is part of the definition of done)